src/docchecks.jl

```diff
@@ -255,6 +265,17 @@ function linkcheck(node::MarkdownAST.Node, doc::Document; method::Symbol=:HEAD)
     return false
 end
 
+
+function linkcheck(node::MarkdownAST.Node, docs_node::Documenter.DocsNode, doc::Document)
+    success = true
+    for mdast in docs_node.mdasts
+        success &= !(linkcheck(mdast, doc) === false)
+        # `linkcheck` may return `true` / `false` / `nothing`
+    end
+    return success
+end
```

Comment:
Is it actually necessary to keep track of `success` here, since we're kinda discarding the information anyway in the main loop?

Reply:
I suppose the options are:

1. Always return `nothing` (i.e., no `return` statements). That works fine (since we discard the information in the main loop, as you say), but is a little "cosmetically" inconsistent with other `linkcheck` methods, in that some of the methods will return `false` on failure and others won't. Also, returning something informative might help in the context of (future) unit tests.
2. Return `false` early as soon as on of the elements in `mdasts` has a failing link. That's probably most consistent with the other `linkcheck` methods, but it seemed better to get a list of *all* broken links in one go, as opposed to having to iterate the edit-push cycle multiple times.
3. The above, keeping track of `success`

So I'd got with option 1 or 3 over 2, but let me know which you'd prefer

Reply:
Well, option 1 actually seems fine for the moment. If someone wants "informative returns" for better unit testing in the future, it can be re-added. Also, in that case we'd probably want to consistently return `true`/`false`, everywhere, not `nothing`/`false`. So, I'll just got with that…